rust/operator-binary/src/authentication/oidc_.rs

```diff
@@ -0,0 +1,152 @@
+use std::collections::BTreeMap;
+
+use snafu::ResultExt;
+use stackable_druid_crd::{
+    security::add_cert_to_jvm_trust_store_cmd, DruidRole, COOKIE_PASSPHRASE_ENV,
+};
+use stackable_operator::{
+    builder::pod::{container::ContainerBuilder, PodBuilder},
+    commons::authentication::oidc::{
+        AuthenticationProvider, ClientAuthenticationOptions, DEFAULT_OIDC_WELLKNOWN_PATH,
+    },
+    k8s_openapi::api::core::v1::EnvVar,
+};
+
+use crate::{
+    authentication::{AddOidcVolumesSnafu, CreateOidcEndpointUrlSnafu, Error},
+    internal_secret::env_var_from_secret,
+};
+
+fn add_authenticator_config(
+    provider: &AuthenticationProvider,
+    oidc: &ClientAuthenticationOptions,
+    config: &mut BTreeMap<String, Option<String>>,
+) -> Result<(), Error> {
+    let endpoint_url = &provider
+        .endpoint_url()
+        .context(CreateOidcEndpointUrlSnafu)?;
+
+    const PREFIX: &str = "druid.auth.pac4j";
```

Comment:
Just nitpicking and under the assumption that the prefix wont change in the future: I prefer copy pasting here to have the full property name (better for search, search / replace and general readability).